autoscheduler/frontend/templates/index.html

```diff
@@ -50,9 +50,13 @@
         <div id="root"></div>
 
         <!-- Dependencies -->
-        <!-- I think we need these? -->
-        <script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/react/16.12.0/umd/react.development.js"></script>
-        <script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.11.0/umd/react-dom.development.js"></script>
+        {% if IS_GCP %}
+            <script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/react/16.12.0/umd/react.production.min.js"></script>
+            <script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/react-dom/16.11.0/umd/react-dom.production.min.js"></script>
```

Comment:
So did you determine that we actually do need these?

Reply:
Yeah, with another project I was working on react imports weren't necessary but we do need them with this node/webpack setup